python/pyspark/sql/dataframe.py

```diff
@@ -1746,7 +1746,7 @@ def toPandas(self):
             pdf = pd.DataFrame.from_records(self.collect(), columns=self.columns)
 
             for f, t in dtype.items():
-                pdf[f] = pdf[f].astype(t, copy=False)
```

Comment:
@cloud-fan, @viirya and @BryanCutler, while I was testing this, I realised that actually it looks `copy` is exposed from 0.13.0. I was confused that it was added from 0.11.0 - [here](https://github.com/pandas-dev/pandas/blob/v0.11.0/pandas/core/generic.py#L521). However, it sounds hidden from 0.11.0 to 0.12.0.

What do you think about this? I think it sounds safer to not use it for now (as I found the doc says we should be careful) and we can support Pandas 0.11.0 and 0.12.0. 

It is still not a big deal maybe. 0.13.0 was released 3.5 years ago. Please let me know. I can just fix the version to 0.13.0.